homeassistant/components/dsmr/sensor.py

```diff
@@ -387,6 +423,113 @@ class MbusDeviceType(IntEnum):
         device_class=SensorDeviceClass.GAS,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    DSMRSensorEntityDescription(
+        key="actual_threshold_electricity",
+        translation_key="actual_threshold_electricity",
+        obis_reference="ACTUAL_TRESHOLD_ELECTRICITY", # Mispelt in external tool
```

Comment:
```suggestion
        obis_reference="ACTUAL_TRESHOLD_ELECTRICITY", # Misspelled in external tool
```

Reply:
Ps. First wanted it reverted, but then changed my mind and just corrected the spelling of the comment.